alonet/raft/raft.py

```diff
@@ -37,20 +37,23 @@ class RAFTBase(nn.Module):
     corr_levels = None
     corr_radius = None
 
+    out_plane = None
```

Comment:
Just curious. Why corr_levels, corr-radius and out_plane are create as static None variable  ? Usually static variable are defined as constant and are != None